Directory.Packages.props

```
@@ -14,6 +14,9 @@
     <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'netstandard2.0'">16.8.0</MicrosoftBuildVersion>
     <!-- The last package version of MSBuild that works with net5.0 is 16.11.0.  Our assemblies are still compiled against MSBuild assembly version 15.1.0.0 which will work with all versions -->
     <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp5.0'">16.11.0</MicrosoftBuildVersion>
+    
+    <!-- Overridden by source build to ensure the same version is used across products, do not remove this property -->
+    <MicrosoftWebXdtPackageVersion>3.0.0</MicrosoftWebXdtPackageVersion>
```

@crummel since you worked on the central package version management stuff before, do you know if this will be called before or after we override the versions?

If this is defined afterwards, we'd need the following change, so we don't override the source-build package version:

```suggestion
    <MicrosoftWebXdtPackageVersion Condition="'$(MicrosoftWebXdtPackageVersion)' == ''">3.0.0</MicrosoftWebXdtPackageVersion>
```

---

Oh good point, I assumed global properties were used, I'll put a condition